packages/tx-ingestion/test/sequencer-batch-append.spec.ts

```diff
@@ -10,11 +10,11 @@ import { ganache } from '@eth-optimism/ovm-toolchain'
 import { OptimismProvider } from '@eth-optimism/provider'
 import { deployContract } from 'ethereum-waffle'
 import assert = require('assert')
-import ERC20 = require('../contracts/build/ERC20.json')
+import ERC20 = require('../../../contracts/build/ERC20.json')
```

Comment:
This file definitely needs to be built out so that it tests sequencer batch append but that is out of scope of this PR

Reply:
Got it. I'll keep that in mind